packages/lab/src/spinner/Spinner.tsx

```diff
@@ -64,14 +64,13 @@ export const Spinner = forwardRef<HTMLDivElement, SpinnerProps>(
       disableAnnouncer,
       role = "img",
       className,
-      size = "medium",
+      size = "default",
       id: idProps,
```

Comment:
```suggestion
      id: idProp,
```

Reply:
this was just going to be the new designs and then improv in new pr, but will commit these :)